website/docs/r/managed_lustre_file_system.html.markdown

```diff
@@ -0,0 +1,138 @@
+
+* `subnet_id` - (Required) The resource ID of the Subnet that is used for managing the Azure Managed Lustre file system and for client-facing operations. This subnet should have at least a /24 subnet mask within the VNET's address space. Changing this forces a new resource to be created.
+
+* `zones` - (Required) A list of availability zones for the Azure Managed Lustre File System. Changing this forces a new resource to be created.
+
+* `hsm_setting` - (Optional) A `hsm_setting` block as defined below. Changing this forces a new resource to be created.
+
+* `identity` - (Optional) An `identity` block as defined below. Changing this forces a new resource to be created.
+
+* `key_encryption_key` - (Optional) A `key_encryption_key` block as defined below.
+
+-> **NOTE:** Removing `key_encryption_key` forces a new resource to be created.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Azure Managed Lustre File System.
+
+---
+
+A `maintenance_window` block supports the following:
+
+* `day_of_week` - (Required) The day of the week on which the maintenance window will occur. Possible values are `Sunday`, `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday` and `Saturday`.
+
+* `time_of_day_utc` - (Required) The time of day (in UTC) to start the maintenance window.
+
+---
+
+A `hsm_setting` block supports the following:
+
+* `container_id` - (Required) The resource ID of storage container that is used for hydrating the namespace and archiving from the namespace. Changing this forces a new resource to be created.
+
+* `logging_container_id` - (Required) The resource ID of storage container that is used for logging events and errors. Changing this forces a new resource to be created.
+
+* `import_prefix` - (Optional) The import prefix for the Azure Managed Lustre File System. Only blobs in the non-logging container that start with this path/prefix get hydrated into the cluster namespace. Changing this forces a new resource to be created.
+
+-> **NOTE:** It has to add roles `Contributor` and `Storage Blob Data Contributor` to the Service Principal `HPC Cache Resource Provider` for the Storage Account. See [official docs]( https://learn.microsoft.com/en-us/azure/azure-managed-lustre/amlfs-prerequisites#access-roles-for-blob-integration) for more information.
+
+---
+
+An `identity` block supports the following:
+
+* `type` - (Required) The type of Managed Service Identity that should be configured on this Azure Managed Lustre File System. Only possible value is `UserAssigned`. Changing this forces a new resource to be created.
+
+* `identity_ids` - (Required) A list of User Assigned Managed Identity IDs to be assigned to this Azure Managed Lustre File System. Changing this forces a new resource to be created.
+
+---
+
+A `key_encryption_key` block supports the following:
+
+* `key_url` - (Required) The URL to the Key Vault Key used as the Key Encryption Key. This can be found as `id` on the `azurerm_key_vault_key` resource.
```

Comment:
```suggestion
* `key_url` - (Required) The URL to the Key Vault Key used as the Encryption Key. This can be found as `id` on the `azurerm_key_vault_key` resource.
```

Reply:
Updated